code/modules/projectiles/projectile/energy/_energy.dm

```diff
@@ -11,4 +11,4 @@
 	. = ..()
 
 	ADD_TRAIT(src, TRAIT_FREE_HYPERSPACE_MOVEMENT, INNATE_TRAIT)
-	ADD_TRAIT(src, TRAIT_FREE_HYPERSPACE_MOVEMENT, TRAIT_FREE_HYPERSPACE_SOFTCORDON_MOVEMENT)
+	ADD_TRAIT(src, TRAIT_FREE_HYPERSPACE_SOFTCORDON_MOVEMENT, INNATE_TRAIT)
```

Comment:
This was definitely an error so I fixed it while going over these traits

Reply:
lol